plugins/modules/source_control/gitlab/gitlab_runner.py

```diff
@@ -281,11 +283,11 @@ def update_runner(self, runner, arguments):
     '''
     @param description Description of the runner
     '''
-    def find_runner(self, description, owned=False):
-        if owned:
-            runners = self._runners_endpoint.list(as_list=False)
-        else:
+    def find_runner(self, description, list_all_shared_runners=True):
```

Comment:
I just had this idea :
Since self._runners_endpoint is now used only to call the right list() / all() function, I think it would be better to put the list / all function reference directly into self._runners_endpoint variable, to simplify the module code.

It would avoid to pass list_all_shared_runners variable in all method calls.

What do you think about that @drzraf @felixfontein ?

Reply:
Fine for me.